packages/nodejs/src/__tests__/metrics.test.ts

```diff
@@ -11,14 +9,14 @@ describe("Metrics", () => {
     metrics = new Metrics()
   })
 
-  it("has `Probes` when minutely probes are on", () => {
-    expect(metrics.probes()).toBeInstanceOf(BaseProbes)
+  it("is not stopped when minutely probes are on", () => {
```

Comment:
```suggestion
  it("is running when minutely probes are on", () => {
```

Reply:
Done! Used "runs the probes"/"does not run the probes" as wording instead, because these tests are on the Metrics object, so it is not "it" that is or isn't running.